### ISSUE TITLE
reanimated v3.0.0 null is not an object (evaluating 'value._isReanimatedSharedValue')

### ISSUE THREAD

**Issue by chj-damon** [software-mansion/react-native-reanimated#4580]
### Description

if `Animated.View` children is null, it will report this error: 

```code
null is not an object (evaluating 'value._isReanimatedSharedValue')
```

### Steps to reproduce

clone this repo

change `https://github.com/chj-damon/reanimatedIssueRepo/blob/main/switch/index.js#LL70C22-L70C22` in `renderText` function. 

comment line 69, and uncomment line 70. the error occurs.

### Snack or a link to a repository

https://github.com/chj-damon/reanimatedIssueRepo

### Reanimated version

3.0.0

### React Native version

0.66.5

### Platforms

iOS

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by chj-damon** [comment]
it seems this error only happens on v3.0.0, I tried to upgrade to 3.1.0 and everything works fine.

**Comment by tomekzaw** [comment]
@chj-damon Thanks for submitting this issue, duplicate of #4129, should be fixed with #4128, released in 3.0.1, please upgrade to 3.3.0.
